examples/ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -26,4 +26,8 @@
     <filter root="/conf/core-components-examples/settings/wcm/policies/wcm/foundation/components/responsivegrid/page-header/jcr:content" mode="update"/>
     <filter root="/conf/core-components-examples/settings/wcm/policies/wcm/foundation/components/responsivegrid/page-footer" mode="update"/>
     <filter root="/conf/core-components-examples/settings/wcm/policies/wcm/foundation/components/responsivegrid/page-footer/jcr:content" mode="update"/>
+    <filter root="/content/forms/af/core-forms-components-examples/sample" mode="merge"/>
+    <filter root="/content/dam/formsanddocuments/core-forms-components-examples/sample" mode="merge"/>
+    <filter root="/content/forms/af/core-forms-components-examples/test-fragment" mode="merge"/>
+    <filter root="/content/dam/formsanddocuments/core-forms-components-examples/test-fragment" mode="merge"/>
```

Comment:
This looks incorrect to me. The whole point of `aemcomponents.dev` is to describe components. The page used in the examples is always a SITE page.

While I agree that fragment have a requiremenet to be under `/content/forms/af`, the sample page used is incorrect. Please refer on how text input is demonstrated and try using a fragment inside an existing SITES page, let's not create a PAGE inside `/content/forms/af`

Reply:
The page use for fragment and embed both are SITE pages . e.g. examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/fragment/.content.xml
But since we need to embed an adaptive form & an adaptive form fragment in these SITE pages, so the adaptive form and fragment are placed under /content/forms/af

The text input use case is not relevant here because text input is just a component under a form container in a SITE page. Here these are embed components which need a reference to an adaptive form.